toolkits/slack/evals/eval_chat.py

```diff
@@ -0,0 +1,1020 @@
+import json
+from datetime import timedelta
+from typing import Any
+
+from arcade.sdk import ToolCatalog
+from arcade.sdk.eval import (
+    BinaryCritic,
+    DatetimeCritic,
+    EvalRubric,
+    EvalSuite,
+    ExpectedToolCall,
+    SimilarityCritic,
+    tool_eval,
+)
+
+import arcade_slack
+from arcade_slack.tools.chat import (
+    get_conversation_history_by_id,
+    get_conversation_history_by_name,
+    get_conversation_metadata_by_id,
+    get_conversation_metadata_by_name,
+    get_members_from_conversation_by_id,
+    get_members_from_conversation_by_name,
+    list_conversations_metadata,
+    list_direct_message_channels_metadata,
+    list_group_direct_message_channels_metadata,
+    list_private_channels_metadata,
+    list_public_channels_metadata,
+    send_dm_to_user,
+    send_message_to_channel,
+)
+
+
+class RelativeTimeBinaryCritic(BinaryCritic):
```

Comment:
cool! We should think about the best file location for custom critics

Reply:
moved to a dedicated module at ./arcade_slack/critics.py